src/dhcp_mon.cpp

```diff
@@ -212,19 +237,33 @@ int dhcp_mon_init(int window_sec, int max_count)
  */
 void dhcp_mon_shutdown()
 {
-    event_del(ev_timeout);
-    event_del(ev_sigint);
-    event_del(ev_sigterm);
-    event_del(ev_sigusr1);
+    rx_event_mgr->free();
```

Comment:
Do we need to reclaim the object here as well?

Reply:
Fixed